pkg/registry/apis/dashboard/sub_dto.go

```diff
@@ -88,10 +93,32 @@ func (r *DTOConnector) Connect(ctx context.Context, name string, opts runtime.Ob
 	r.getAnnotationPermissionsByScope(ctx, user, &access.AnnotationsPermissions.Dashboard, accesscontrol.ScopeAnnotationsTypeDashboard)
 	r.getAnnotationPermissionsByScope(ctx, user, &access.AnnotationsPermissions.Organization, accesscontrol.ScopeAnnotationsTypeOrganization)
 
-	dash, err := r.builder.access.GetDashboard(ctx, info.OrgID, name)
+	key := &resource.ResourceKey{
+		Namespace: info.Value,
+		Group:     dashboard.GROUP,
+		Resource:  dashboard.DashboardResourceInfo.GroupResource().Resource,
+		Name:      name,
+	}
+	store := r.builder.legacy.access
+	rsp, err := store.Read(ctx, &resource.ReadRequest{Key: key})
+	if err != nil {
+		return nil, err
+	}
+	dash := &dashboard.Dashboard{}
+	err = json.Unmarshal(rsp.Value, dash)
 	if err != nil {
 		return nil, err
 	}
+
+	obj, err := utils.MetaAccessor(dash)
+	if err != nil {
+		return nil, err
+	}
+	blobInfo := obj.GetBlob()
+	if blobInfo != nil {
+		fmt.Printf("TODO, load full blob from storage %+v\n", blobInfo)
+	}
+
```

Comment:
follow-up PR?

Reply:
yes -- this will require something like [this blob support](https://github.com/grafana/grafana/pull/89071/files#diff-9be6cc59c33d7120ea8fa99ee656adddb88aab45c250799888a73dcffc752a00R469) to continue